lib/ex_aws/cloud_search/document.ex

```diff
@@ -0,0 +1,5 @@
+defprotocol ExAws.CloudSearch.Document do
+  @fallback_to_any true
```

Comment:
IC so the comment was about the `@fallback_to_any`. And you're right, we don't need it.

I will add documentation cause (a) it's nice and (b) it'll explain my choice a little.